doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
-- **Compute flavour**: defines the sizing of the virtualised resources (compute, memory, and storage) required to run a workload.
+
+- **Compute flavour:** defines the sizing of the virtualised resources (compute, memory, and storage) required to run a workload.
      >_*Note:*_ used to define the configuration/capacity limit of a virtualised container.
-- **VM instances Catalogue**: Pre-defined instance types and compute flavours.
-- **Virtual resources**:
-  - **Virtual Compute resource (a.k.a. virtualisation container)**: partition of a compute node that provides an isolated virtualised computation environment.
-  - **Virtual Storage resource**: virtualised non-volatile storage allocated to a virtualised computation environment hosting a **VNFC**
-  - **Virtual Networking resource**: routes information among the network interfaces of a virtual compute resource and physical network interfaces, providing the necessary connectivity
-- **Hypervisor**: A piece of software with management components that allows the user to partition the underlying physical resources and allocate them to Virtual Machines. Typically, the hypervisor is managed by a cloud management software such as OpenStack.
-- **Cloud Infrastructure Profile**: The combination of the Cloud Infrastructure Software Profile and the Cloud Infrastructure Hardware Profile that defines the capabilities of the Cloud Infrastructure.
-- **Cloud Infrastructure Software Profile**: defines the behaviour, capabilities and metrics provided by a cloud infrastructure Software Layer
-- **Cloud Infrastructure Software Configuration**: a set of settings (Key:Value) that are applied/mapped to **cloud infrastructure** SW deployment.
+
+- **Hypervisor:** A piece of software with management components that allows the user to partition the underlying physical resources and allocate them to Virtual Machines. Typically, the hypervisor is managed by a cloud management software such as OpenStack.
+
+- **Instance:** is a virtual compute resource, in a known state such as running or suspended, that can be used like a physical server.
+    >_*Note:*_ Can be used to specify VM Instance or Container Instance.
+
+- **Network Function (NF):**  functional block or application within a network infrastructure that has well-defined external interfaces and well-defined functional behaviour.
+  - Within **NFV**, A **Network Function** is implemented in a form of **Virtualised NF** or a **Cloud Native NF**.
+
+
+- **Network Function Virtualisation (NFV):** The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
+
+- **Network Function Virtualisation Infrastructure (NFVI):** The totality of all hardware and software components used to build the environment in which a set of VAs are deployed (also referred to as cloud infrastructure).
```

Comment:
VA first use; laso minor correction to avoid confusion with the content in parentheses immediately following "VAs"
```suggestion
- **Network Function Virtualisation Infrastructure (NFVI):** The totality of all hardware and software components used to build the environment in which a set of virtual applications (VAs) are deployed; also referred to as cloud infrastructure.
```